src/features/common/Layout/Navbar/index.tsx

```diff
@@ -130,6 +130,17 @@ export default function NavbarComponent(props: any) {
                 </button>
               )
             }
+            if (link === 'about' && SingleLink.visible) {
+              return (
+                <button id={'aboutUsIcon'} key={link} onClick={() => router.push(`${SingleLink.onclick}${i18n.language === 'de' ? 'de' : ''}`,undefined,undefined)} className={'linkContainer'}>
```

Comment:
This seems to work going to https://a.plant-for-the-planet.org/de instead of https://a.plant-for-the-planet.org/ for German language, but are wonder if Wordpress is not able to detect the current browser language itself and show the right localized page to our users as we do for the website?

Reply:
This is intended, Wordpress team has sent this request.